uxarray/utils/constants.py

```diff
@@ -3,3 +3,5 @@
 # numpy indexing code is written for np.intp
 INT_DTYPE = np.intp
 INT_FILL_VALUE = np.iinfo(INT_DTYPE).min
+ENABLE_JIT_CACHE = True
+ENABLE_JIT = False
```

Comment:
Should this instead directly modify:

```Python
config.DISABLE_JIT
```

Reply:
Isn't that what we are doing in the helpers.py and quadaturesDG.py files? We can't do it inside the constants file unless we want to import numba into that file which doesn't make a whole lot of sense.

Reply:
> unless we want to import numba into that file

Didn't consider this, the current implement is good then!